src/viser/extras/_urdf.py

```diff
@@ -71,16 +74,24 @@ def __init__(
         urdf_or_path: yourdfpy.URDF | Path,
         scale: float = 1.0,
         root_node_name: str = "/",
-        mesh_color_override: tuple[float, float, float] | None = None,
+        show_visuals: bool = True,
+        show_collisions: bool = False,
+        collision_mesh_color_override: tuple[float, float, float] | None = None,
+        visual_mesh_color_override: tuple[float, float, float] | None = None,
```

Comment:
can we avoid moving / renaming this argument? 

it seems like this would also work:
```python
        mesh_color_override: tuple[float, float, float] | None = None,
        collision_mesh_color_override: tuple[float, float, float] | None = None,
        show_visuals: bool = True,
        show_collisions: bool = False,
```